pkg/controller/oneagent/istio.go

```diff
@@ -13,13 +14,26 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
 func (r *ReconcileOneAgent) reconcileIstio(logger logr.Logger, instance *dynatracev1alpha1.OneAgent, dtc dtclient.Client) error {
 	var err error
 
-	// TODO determine if cluster runs istio
+	cfg, err := config.GetConfig()
+	if err != nil {
+		logger.Error(err, "error getting default config")
+		return err
+	}
+
+	// determine if cluster runs istio in default cluster
+	exists, err = istio.CheckIstioEnabled(cfg)
+	if err != nil {
+		logger.Error(err, "error checking for istio enabled")
+		return err
+	}
+	logger.Info(fmt.Sprintf("istio enabled : %v", exists))
```

Comment:
Something like `logger.Info("looked for Istio", "enabled", exists)`

Reply:
I think there needs to be a case handling here, will double check with markus and implement.